templates/loop/rating.php

```diff
@@ -21,8 +21,8 @@
 
 global $product;
 
-if ( get_option( 'woocommerce_enable_review_rating' ) === 'no' ) {
+if ( 'no' === get_option( 'woocommerce_enable_review_rating' ) || 'no' === get_option( 'woocommerce_enable_reviews' ) ) {
```

Comment:
Can we make a helper function for this to see if ratings should be enabled/visible? Then if we need to tweak this logic in the future, we can avoid a template update.